packages/types/src/interfaces/injects.ts

```diff
@@ -33,3 +34,25 @@ const _types = {
 
 // Unwind the nested type definitions into a flat map
 export default Object.values(_types).reduce((res, { types }): object => ({ ...res, ...types }), {});
+
+export const typesBundle: OverrideBundleType = {
+  spec: {
+    cennznet: {
+      types: [
+        {
+          minmax: [0, 36],
+          types: {
+            ...syloTypes,
+            ExtrinsicSignatureV4: CENNZnetExtrinsicSignatureV0,
+          },
+        },
+        {
+          minmax: [37, 37],
```

Comment:
Can we make this automatically include our future types > 37?

Reply:
I will check if there is any option..

Reply:
https://github.com/polkadot-js/api/blob/98b01a462dfc8d3280396d76c887ff8a3e1cd2e5/packages/types/src/types/registry.ts#L98
The minmax is of type [number, number] and has ? so can be undefined...
https://github.com/polkadot-js/api/blob/2c735fbc5397d482c331ff335c10faadb0adeadc/packages/types-known/src/spec/node-template.ts#L8